src/Components/ArticlePreview/ArticlePreview.js

```diff
@@ -1,19 +1,38 @@
-import React from "react";
-import ArticleExpanded from "../ArticleExpanded/ArticleExpanded";
-import styled from "styled-components";
+import React, {useState} from 'react';
+import ArticleExpanded from '../ArticleExpanded/ArticleExpanded'
+import styled from 'styled-components'
+import {fetchImage} from '../../apiCalls'
 
 const StoryCard = styled.div`
   background-color: purple;
   width: 75%;
   height: 25%;
-`;
+
+  img {
+    width: 20px;
+  }
+`
 
 const ArticlePreview = (props) => {
+
+  const [image, setImage] = useState(null);
+  const [description, setDescription] = useState(null);
+
+  fetchImage(props.url)
+    .then(data => {
+      return (
+        setImage(data.hybridGraph.image),
+        setDescription(data.hybridGraph.description)
+      )
+    })
+
```

Comment:
We'll change this method name maybe to "fetch details"  - this is fetching the details of each URL using the opengraph API.

Reply:
that's a good name, more semantic